Sources/PIRGenerateDatabase/PIRGenerateDatabase.docc/PIRGenerateDatabase.md

```diff
@@ -29,7 +29,7 @@ PIRGenerateDatabase \
 
 This will generate a database of 100 rows, with keywords 0 to 100, and each value repeating the keyword for 10 to 20 bytes.
 
-The database is a serialized `KeywordDatabase` message.
+The database is a serialized [KeywordDatabase](https://swiftpackageindex.com/apple/swift-homomorphic-encryption/main/documentation/privateinformationretrieval/keyworddatabase) message.
```

Comment:
Maybe link to https://swiftpackageindex.com/apple/swift-homomorphic-encryption/main/documentation/privateinformationretrievalprotobuf/apple_swifthomomorphicencryption_pir_v1_keyworddatabase ?